src/tests/ftest/pool/destroy_rebuild.yaml

```diff
@@ -14,13 +14,15 @@ hosts:
 server_config:
   name: daos_server
   servers:
-    targets: 8
+    targets: 2
 pool:
   mode: 511
   name: daos_server
-  scm_size: 16428800
+  scm_size: 134217728
   control_method: dmg
+  pool_query_timeout: 30
 testparams:
-  rank_to_kill: 0
-  # rank_to_kill: 1
-  # rank_to_kill: 3
+  rank_to_kill:
```

Comment:
Remove comment here since it is skipped via cancelforticket?

Reply:
So I think we can turn this into mux:
testparams:
  ranks: !mux
    rank1:
      rank: 1
    rank2:
      rank: 2
    rank3:
      rank: 3

Reply:
Not turning into mux as the intent of the test was to stop multiple ranks together.